omniledger/service/service.go

```diff
@@ -323,5 +344,12 @@ func newService(c *onet.Context) (onet.Service, error) {
 		log.Error(err)
 		return nil, err
 	}
+
+	var err error
+	s.propagateTransactions, err = messaging.NewPropagationFunc(c, "OmniledgerPropagate", s.updateCollection, -1)
+	if err != nil {
+		return nil, err
+	}
+	s.propTimeout = 10 * time.Second
```

Comment:
Who would be changing that? I think it can be for tests, but it might also have to be changed for some special implementation... 

I added a `Service.SetPropagationTimeout`.